### ISSUE TITLE
Blocky charts

### ISSUE THREAD

**Issue by asymmetric** [chubin/rate.sx#21]
Hi,

There must be something wrong with my terminal configuration (maybe the fonts?), because this is how charts render:

![2018-12-25-121202_1920x1080_scrot](https://user-images.githubusercontent.com/101816/50421339-5b0bb800-083e-11e9-8940-f85969896b61.png)

I'm using alacritty with the [hack font](https://sourcefoundry.org/hack/).

Let me know if there's any other config that could help debug the issue.

**Comment by chubin** [comment]
@asymmetric Yes, Lorenzo, it is terminal font. Please try to start `uxterm` and check if it works there.

**Comment by asymmetric** [comment]
Yes, on `uxterm` it looks as expected, thanks! Is there a way to know which/why fonts look bad?

Btw, if I try the characters `⢄⢠⡄` [here](https://sourcefoundry.org/hack/playground.html), they seem to look fine.

**Comment by chubin** [comment]
Does this work for you?
(probably not):
```
printf '[\xe2\xa0\x89\xe2\xa0\x9b\xe2\xa0\xbf\xe2\xa3\xbf]\n'
```

**Comment by chubin** [comment]
And what about this:
```
pango-view --font 'Ubuntu Mono 12' --text $'[\xe2\xa0\x89\xe2\xa0\x9b\xe2\xa0\xbf\xe2\xa3\xbf]\n'
```

(your font instead of `Ubuntu Mono 12`)
